client/state/account-recovery/test/test-data.js

```diff
@@ -0,0 +1,13 @@
+const dummyData = {
+	email: 'dummytest@a8ctest.com',
```

Comment:
Is `a8ctest.com` a domain under our control? Even for test data I would suggest avoiding anything that could lead someone somewhere nasty. Consider `example.com` instead (an IANA reserved domain).